app/controllers/admin/loans_controller.rb

```diff
@@ -10,8 +10,18 @@ def index
       order: 'loans.signing_date',
       order_direction: 'desc',
       custom_order: { 'loans.signing_date' => 'loans.signing_date IS NULL, loans.signing_date' },
-      per_page: 50
+      per_page: 50,
+      name: 'loans',
+      enable_export_to_csv: true,
+      csv_file_name: 'loans'
     )
+
+    @csv_mode = true
+
+    export_grid_if_requested('loans' => 'loans_grid') do
```

Comment:
I don't think you need the argument here. I think loans -> loans_grid is the default. Let me know if I'm wrong. In general the Rails philosophy is convention over configuration so good to stick to that if possible.